packages/mux-player/src/template.ts

```diff
@@ -37,7 +30,7 @@ const getHotKeys = (props: MuxTemplateProps) => {
 
 export const content = (props: MuxTemplateProps) => html`
   <media-theme
-    template="${props.themeTemplate ?? muxTemplate.content.children[0]}"
+    template="${props.themeTemplate ?? classicTheme.template}"
```

Comment:
this could be removed ` ?? classicTheme.template` then if you make `theme` prop `classic` by default